client/src/components/PopUp/index.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
Add a new line after import third-party modules and then put import the style file  
```suggestion
import React from "react";
import Dialog from "@material-ui/core/Dialog";
import DialogActions from "@material-ui/core/DialogActions";
import Button from "./../Button";
import DialogTitle from "@material-ui/core/DialogTitle";

import "./style.css";
```